lab_1/main.py

```diff
@@ -69,7 +114,53 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    # bad_inputs = ['string', (), None, 9, 9.34, True, [None], []]
+    concordance = []
+    if isinstance(tokens, list):
+        for t in tokens:
+            if not isinstance(t, str):
+                break
+    if (isinstance(tokens, list)):
```

Comment:
seems like there are two such checks
maybe write one check at the beginning which will allow you to do everything without checking if "tokens" is a list?